harness/determined/cli/master.py

```diff
@@ -65,6 +77,10 @@ def process_response(response: Response, latest_log_id: int) -> int:
             Arg("-o", "--output", type=str, default="yaml",
                 help="Output format, one of json|yaml")
         ]),
+        Cmd("info", get_master, "fetch master info", [
+            Arg("-o", "--output", type=str, default="yaml",
```

Comment:
would you say a two-letter flag with a single is `-` also uncommon?

Reply:
yes

Reply:
we seem to be using `--output` elsewhere in the cli as well but not for `--format`. would it be a breaking change? if so would it be better to tackle that in a separate pr

Reply:
Only in one command.  I'd argue that command is wrong, and copying it doesn't make sense because far more cli commands use `--json` for json formatting.  If you have to copy a style I'd prefer to copy that one.

Also I realize now that `-f` is frequently used in our cli for `--follow`, so actually I think `--json` is a fine idea here.

But, yeah our cli options are a total mess

Reply:
I [updated](https://github.com/determined-ai/determined/pull/3361/commits/7f0d3de48b02c21c92bfcb1d44acd7afe6d76cfe) the command arguments added in this pr. we can update `-rp` and `--output` for the existing ones separately as they are breaking changes.